From 647c4796341069c0b8b4ccc397c85df66bbe7d69 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Sat, 26 Sep 2015 17:43:10 +0800
Subject: [PATCH 983/987] powerpc/t208x/dts: add qman-fqd/qman-pfdr memory node
 for t208x

According to Documentation/devicetree/bindings/soc/fsl/qman.txt, we add
qman-fqd/qman-pfdr memory node for t208x, or it will occur compile error
as below:

ERROR (phandle_references): Reference to non-existent node or label "qman_fqd"
ERROR (phandle_references): Reference to non-existent node or label "qman_pfdr"

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/powerpc/boot/dts/t208xqds.dtsi | 12 ++++++++++++
 arch/powerpc/boot/dts/t208xrdb.dtsi | 12 ++++++++++++
 2 files changed, 24 insertions(+)

diff --git a/arch/powerpc/boot/dts/t208xqds.dtsi b/arch/powerpc/boot/dts/t208xqds.dtsi
index b1169fc..0115367 100644
--- a/arch/powerpc/boot/dts/t208xqds.dtsi
+++ b/arch/powerpc/boot/dts/t208xqds.dtsi
@@ -48,6 +48,18 @@
 			size = <0 0x1000000>;
 			alignment = <0 0x1000000>;
 		};
+		qman_fqd: qman-fqd {
+			compatible = "fsl,qman-fqd";
+			alloc-ranges = <0 0 0x10 0>;
+			size = <0 0x400000>;
+			alignment = <0 0x400000>;
+		};
+		qman_pfdr: qman-pfdr {
+			compatible = "fsl,qman-pfdr";
+			alloc-ranges = <0 0 0x10 0>;
+			size = <0 0x2000000>;
+			alignment = <0 0x2000000>;
+		};
 	};
 
 	ifc: localbus@ffe124000 {
diff --git a/arch/powerpc/boot/dts/t208xrdb.dtsi b/arch/powerpc/boot/dts/t208xrdb.dtsi
index cd5d23b..e30b1b76 100644
--- a/arch/powerpc/boot/dts/t208xrdb.dtsi
+++ b/arch/powerpc/boot/dts/t208xrdb.dtsi
@@ -48,6 +48,18 @@
 			size = <0 0x1000000>;
 			alignment = <0 0x1000000>;
 		};
+		qman_fqd: qman-fqd {
+			compatible = "fsl,qman-fqd";
+			alloc-ranges = <0 0 0x10 0>;
+			size = <0 0x400000>;
+			alignment = <0 0x400000>;
+		};
+		qman_pfdr: qman-pfdr {
+			compatible = "fsl,qman-pfdr";
+			alloc-ranges = <0 0 0x10 0>;
+			size = <0 0x2000000>;
+			alignment = <0 0x2000000>;
+		};
 	};
 
 	ifc: localbus@ffe124000 {
-- 
1.9.1

